### ISSUE TITLE
On scrolling up and down in the notes list, stuttering movement

### ISSUE THREAD

**Issue by LNFWebsite** [federicoiosue/Omni-Notes#438]
When scrolling through the notes list quickly, the movement appears to slightly stutter. This makes it slightly harder to read and choose a note from the list.

Thanks for developing such a cool notes app!

**Comment by federicoiosue** [comment]
Could you please provide more informations by writing your device and attaching a screenshot of your **Statistics** section into settings please, to understand the entity of data you're managing?

**Comment by LNFWebsite** [comment]
Device: Moto X Pure Edition (2015) XT1575
OS: Android Nougat 7.0 (not rooted)

![screenshot_20180410-150635](https://user-images.githubusercontent.com/9419359/38578190-36b8139c-3cd1-11e8-93f3-844659d94e9b.png)

If I had to guess, I think that your interface for selecting notes is always prepping for swipe motions or is skipping around because of the add note button that disappears. Something seems to cause the interface to quickly and stutteringly re-render on scroll.

Hope this helps!

**Comment by LNFWebsite** [comment]
Have you gotten to the bottom of the issue or are you scraping it in favor of bigger tasks to work on? I understand either way; being an dev is hard.

Just curious as what I can expect as a user. Thanks

**Comment by dubhater** [comment]
Hi.

I have the same problem with the stuttering when scrolling the list of notes.

Additionally, I noticed another animation that stutters: when I close a (text) note the Back button with the arrow pointing left in the top left corner transitions into the button with three horizontal lines (I don't know the name). That transition is not smooth. The arrow turns a few degrees (less than 45), freezes, and a moment later it's replaced with the three horizontal lines.

When I'm in the list of notes and I open a note, the button with the three horizontal lines transitions smoothly into the Back button with the arrow.

I installed Omni Notes 6.0.4 from the Play Store. Here are the statistics:
![Screenshot_20191112-152150](https://user-images.githubusercontent.com/1739488/68675654-00b43c80-0561-11ea-837f-d57bdd15535b.png)
![Screenshot_20191112-152158](https://user-images.githubusercontent.com/1739488/68675653-00b43c80-0561-11ea-8e09-a3ca1c79c76c.png)

My device is a Myria Five with Android 7.0. Here are some of [the specifications](http://myria.ro/en/smartphone/1858/myria-five-my9039-smartphone-dual-sim-5-8mp-2gb-ram-16gb-quad-core-4g-black.html):
```
Screen
Screen type                         Capacitive touchscreen, HD
Screen size (inch)                  5
Screen resolution (pixeli)          720 x 1280

Processor
Processor type                      Quad-core
Processor model                     MTK6737
Frecventa (GHz)                     1.3
Graphic processor                   Mali720-MP1

Memory
Internal memory                     16 GB
RAM memory                          2 GB
Card slot                           microSD
Memory card maximum capacity (GB)   32

Software
Operating system                    Android, V7.0
```

**Comment by dubhater** [comment]
Hi!

Disabling „Prettified dates” fixes the stuttering for me. Both instances of stuttering I described above.

**Comment by federicoiosue** [comment]
Hi @dubhater, did you also tried the latest 6.1.0? Because, also setting my device on  battery-saver mode (that slows down performances) and scrolling a list of many hundred notes I had no particular issues both with "Prettified Dates" or without it activated

**Comment by dubhater** [comment]
No, I haven't received that update yet. I'm still using 6.0.5.

**Comment by federicoiosue** [comment]
Now you should have received update because I've completed the progressive release.

**Comment by dubhater** [comment]
You fixed it! 6.1.0 doesn't have the stuttering, even with the prettified dates enabled. Thank you!

**Comment by federicoiosue** [comment]
![bitmoji](https://sdk.bitmoji.com/render/panel/7b6c1ca1-98da-480d-a8ac-768c8ce4dd24-60b8a5dc-65eb-468b-a8ee-e63c10a1dc4c-v1.png?transparent=1&palette=1&width=246)
